static/src/stylesheets/module/content/_badging.scss

```diff
@@ -61,3 +88,39 @@
     }
 }
 
+/* AUS Election Liveblog Image
+   ========================================================================== */
+.content__head--aus-election {
```

Comment:
Agreed I wanted to just get something up then de dupe and make more generic in a separate thing but i guess i can do it here.